packages/accordions/src/elements/stepper/Stepper.tsx

```diff
@@ -40,6 +40,8 @@ interface IStepperProps extends HTMLAttributes<HTMLOListElement> {
  *  - `Stepper.Step`
  *  - `Stepper.Label`
  *  - `Stepper.Content`
+ * 
+ * @extends HTMLOListElement
```

Comment:
Should be `HTMLAttributes<HTMLOListElement>`

Reply:
This may need to be more specific with `OlHTMLAttributes<HTMLOListElement>`

Reply:
Fixed with Austin's type suggestion.

Reply:
extends have been removed for now.